File: C++/Básico/Operators.cpp
#include <iostream>

int main() {
    int a = 10, b = 3;

    // Operadores aritméticos
    int soma = a + b;        // Soma
    int subtracao = a - b;   // Subtração
    int multiplicacao = a * b; // Multiplicação
    int divisao = a / b;     // Divisão inteira
    int resto = a % b;       // Resto da divisão

    // Operadores relacionais
    bool igual = (a == b);   // Igualdade
    bool diferente = (a != b); // Diferença
    bool maior = (a > b);    // Maior que
    bool menor = (a < b);    // Menor que
    bool maiorIgual = (a >= b); // Maior ou igual
    bool menorIgual = (a <= b); // Menor ou igual

    // Operadores lógicos
    bool e_logico = (a > 5 && b < 5); // E lógico
    bool ou_logico = (a < 5 || b < 5); // OU lógico
    bool nao_logico = !(a == b);      // NÃO lógico

    std::cout << "Soma: " << soma << std::endl;
    std::cout << "Subtração: " << subtracao << std::endl;
    std::cout << "Multiplicação: " << multiplicacao << std::endl;
    std::cout << "Divisão: " << divisao << std::endl;
    std::cout << "Resto: " << resto << std::endl;

    std::cout << "Igual: " << igual << std::endl;
    std::cout << "Diferente: " << diferente << std::endl;
    std::cout << "Maior: " << maior << std::endl;
    std::cout << "Menor: " << menor << std::endl;
    std::cout << "Maior ou igual: " << maiorIgual << std::endl;
    std::cout << "Menor ou igual: " << menorIgual << std::endl;

    std::cout << "E lógico: " << e_logico << std::endl;
    std::cout << "OU lógico: " << ou_logico << std::endl;
    std::cout << "NÃO lógico: " << nao_logico << std::endl;

    return 0;
}

File: PHP/Básico/Operators.php
<?php
// Operadores em PHP

$a = 10;
$b = 3;

// Operadores aritméticos
echo "Soma: " . ($a + $b) . "<br>";
echo "Subtração: " . ($a - $b) . "<br>";
echo "Multiplicação: " . ($a * $b) . "<br>";
echo "Divisão: " . ($a / $b) . "<br>";
echo "Módulo: " . ($a % $b) . "<br>";

// Operadores de atribuição
$a += 5; // $a = $a + 5
echo "Atribuição (+=): $a<br>";

// Operadores de comparação
$x = 5;
$y = "5";
echo "Igual (==): " . ($x == $y ? "true" : "false") . "<br>";
echo "Idêntico (===): " . ($x === $y ? "true" : "false") . "<br>";
echo "Diferente (!=): " . ($x != $y ? "true" : "false") . "<br>";
echo "Maior que: " . ($a > $b ? "true" : "false") . "<br>";

// Operadores lógicos
$verdadeiro = true;
$falso = false;
echo "E lógico (&&): " . ($verdadeiro && $falso ? "true" : "false") . "<br>";
echo "Ou lógico (||): " . ($verdadeiro || $falso ? "true" : "false") . "<br>";
echo "Negação (!): " . (!$verdadeiro ? "true" : "false") . "<br>";

// Operador ternário
$resultado = ($a > $b) ? "A é maior" : "B é maior ou igual";
echo "Ternário: $resultado<br>";
?>